pkg/version/version.go

```diff
@@ -62,3 +70,17 @@ func Version() (string, error) {
 	}
 	return releaseName, nil
 }
+
+// DefaultArch returns the default release architecture
+func DefaultArch() types.Architecture {
+	switch defaultArch {
+	case "arm64":
+		return types.ArchitectureARM64
+	case "ppc64le":
+		return types.ArchitecturePPC64LE
+	case "s390x":
+		return types.ArchitectureS390X
+	default:
+		return types.ArchitectureAMD64
+	}
```

Comment:
Are the values of defaultArch different than the values of types.Architecture? We want to end up with a validation error when an unknown `defaultArch` is used, so we don't want to default in this function to amd64.
```suggestion
	return types.Architecture(defaultArch)
```